.travis.yml

```diff
@@ -2,15 +2,17 @@ language: python
 
 python:
   - 2.7
+  - 3.6
 
 env:
-  - TOXENV=django18
   - TOXENV=django111
 
 matrix:
   include:
-    - python: 2.7
+    - python: 3.6
       env: TOXENV=quality
+    - python: 3.6
+      env: TOXENV=django20
```

Comment:
Maybe add another entry for Django 2.1 now that it's supported in tox?

Reply:
Agreed - done.